katas/content/index.json

```
@@ -9,6 +9,7 @@
   "preparing_states",
   "single_qubit_measurements",
   "multi_qubit_measurements",
+  "superdense_coding",
   "random_numbers",
```

Let's move this one after QRNG, that's usually how these applications are taught, and that matches the order in the classic katas. (Teleportation will be between these two)
```suggestion
  "random_numbers",
  "superdense_coding",
```

---

Done. Thanks for your suggestion.